### PR TITLE
267 solr cloud

### PR DESCRIPTION
**Pull request by cdmo**
To test this locally, you'll need to also checkout the branch being PR'd at https://github.com/psu-libraries/psulib_traject/pull/144

Then:

1. kill felix
2. run the Procfile.dev
3. run `bundle exec rails docker:build` (uploads the config to the embedded zk and creates a collection referencing that)

I had to add a manual extra step for config and collection because `docker run` never finishes execution. So, any tasks that happen after that one don't get called. System calls are blocking.

After this then you can just go back to running that procfile

### CODE REVIEW

**Comment by cdmo** [comment]
I didn't bother figuring out how to make the collection name a global constant...

File: config/blacklight.yml [hunk 1]
```diff
@@ -12,10 +12,10 @@
 
 development:
   adapter: solr
-  url: <%= ENV['SOLR_URL'] || "http://127.0.0.1:8983/solr/blacklight-core" %>
+  url: <%= ENV['SOLR_URL'] || "http://127.0.0.1:8983/solr/psul_blacklight" %>
```
Comment (banukutlu) [inline config/blacklight.yml]: I like psul_blacklight 👌

**Comment by banukutlu** [comment]
worked on my local on the first trial 👍

File: .travis.yml [hunk 2]
```diff
@@ -13,12 +13,11 @@ env:
...
-before_install:
-- docker pull solr:7.4.0
-- docker run -d -p 8983:8983 -v "$(pwd)"/solr/conf:/myconfig solr:7.4.0 solr-create -c blacklight-core -d /myconfig
-- docker ps -a
-
 before_script:
+- bundle exec rails docker:pull
+- bundle exec rails docker:run_ci
+- bundle exec rails docker:build
+- docker ps -a
```
Comment (banukutlu) [inline .travis.yml]: @cdmo do we need this? it just displays the running container, right?

Comment (cdmo) [reply .travis.yml]: it has been there all along... not sure why it's showing it as added. But, anyway, it's not really needed no :)

**Pull request merged**